haproxy/tox.ini

```diff
@@ -29,8 +29,9 @@ setenv =
   18: HAPROXY_VERSION=1.8.27
   20: HAPROXY_VERSION=2.0.20
   21: HAPROXY_VERSION=2.1.11
-  22: HAPROXY_VERSION=2.2.8
-  23: HAPROXY_VERSION=2.3.4
+  22: HAPROXY_VERSION=2.2.9
+  23: HAPROXY_VERSION=2.3.5
```

Comment:
Why these two bumps?

Reply:
it is always better to be close to the last minor available